src/Catty/Helpers/Util/Util.swift

```diff
@@ -70,6 +98,32 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         return deviceName
     }
 
+    class func look(withName objectName: String?, forObject object: SpriteObject?) -> Look? {
```

Comment:
This method makes only sense when providing an name and the object itself, thus I would suggest to make those parameters non-optional. Please also consider making a more Swift-like name (while the Objective C name is perfect, `@objc`), .e.g., `look(with name: String, for object: SpriteObject?) -> Look`. PS: `objectName` is confusing not named in the right context -- I think that parameter refers to the name of the look, right?

Reply:
I've changed the method signature following your suggestions, and yes you are right `objectName` was referring to the name of the look in this case and this really was confusing.